prepare_elmer.bash

```diff
@@ -24,6 +24,23 @@ if [ ! -d SELMER  ]; then mkdir -p  $SELMER   ; fi
 # to fix issue for vtu group gidbit
 chgrp -R ${GROUPUSR} ${WELMER}
 
+# Calculate time offsets of forcing files
+function value_from_incf_file {
+    # Print value (from incf file) of variable given as argument.
+    echo $(grep -E "^[ \t]*\\\$$1[ \t]*=" $CONFIG-${CASE}_elmer.incf | cut -d= -f2 | cut -d! -f1 | cut -d\" -f2)
+}
+function start_year_of_netcdf_file {
+    # Print year of first time frame in netcdf file (given the name of the file in incf file).
+    typeset filepath=$(value_from_incf_file data_dir)/$(value_from_incf_file $1)
+    # For now, we rely on the netcdf file having a properly defined time
+    # variable. Other methods can be implemented later on a need basis
+    echo $(ncdump -t -v time $filepath | grep -E "^ time = \"" | cut -d\" -f2 | cut -d- -f1)
```

Comment:
Is the 'time' variable a requirement by elmer on the input file ? If yes, that's fine. If no, we should add a sanity check on 'time' variable presence. 

I could suggest this as workaround (test avec un fichier bidon que j'avait sous la main):
```
ncdump -v time_centeredsdf eORCA1.L121-OPM003_1d_gridT_0003.nc > /dev/null 2>&1 || `ncdump -t -v time_centeredsdf eORCA1.L121-OPM003_1d_gridT_0003.nc 2> /dev/null | grep -E "^ time_centeredsdf = \"" | cut -d\" -f2 | cut -d- -f1` && echo -99
```
It will output -99 if the variable does not exist. A test can then be added in line 43.

Reply:
I made a commit to add the sanity check as suggested (implemented a but differently though).